docs/_ug/ItemsTags.md

```diff
@@ -14,15 +14,15 @@ The following are the attributes stored for each item:
 * Item remark
 * Item tags
 
-FoodRem items are unique by name and case-insensitive. 
+FoodRem items are unique by name and case-sensitive. 
```

Comment:
Are they case sensitive? I thought `potato` and `Potato` would be considered the same item...

Reply:
Let me double-check with the rest. Based on our current implementation on master, `Potato` and `potato` are treated as different items.

Reply:
Ok, current implementation is case-sensitive. So, on the UG, we'll write it as case-sensitive first. Should we (and when we) update the implementation we can change this

Reply:
Noted, thanks!